rbac/server/api/proposals.py

```diff
@@ -145,6 +145,16 @@ class ProposalType(object):  # pylint: disable=too-few-public-methods
     )
 
 
+@PROPOSALS_BP.patch("api/proposals")
+@authorized()
+async def batch_update_proposals(request):
```

Comment:
Can you add a doc string to this so we know what it's for?

Reply:
Fixed. 😺